solarforecastarbiter/datamodel.py

```diff
@@ -419,6 +419,130 @@ class Observation(BaseModel):
     __post_init__ = __set_units__
 
 
+@dataclass(frozen=True)
+class AggregateObservation(BaseModel):
+    """
+    Class for keeping track of Observations in Aggregates
+    and when they are added and removed from the Aggregate
```

Comment:
Description made me think this class would concern itself with multiple Observations. Suggest:

Class for keeping track of an Observation and when it is added and (optionally) removed from an Aggregate. This metadata allows the Arbiter to calculate the correct quantities while the Aggregate grows or shrinks over time.

Reply:
good suggestion